variants/esp32c6/pins_arduino.h

```diff
@@ -32,4 +32,9 @@ static const uint8_t A4 = 4;
 static const uint8_t A5 = 5;
 static const uint8_t A6 = 6;
 
+// LP I2C Pins are fixed on ESP32-C6
+#define WIRE1_PIN_DEFINED
```

Comment:
```suggestion
#define WIRE1_PINS_DEFINED
```

Reply:
See https://github.com/espressif/arduino-esp32/pull/10858/files#diff-90b03e94086217edc0bf5abd688554e4165b4d7401b8413dc18c27e3263378d8R75
I used the same define as original.